src/components/Search/SearchAutocompleteList.tsx

```diff
@@ -523,6 +523,9 @@ function SearchAutocompleteList(
 
     return (
         <SelectionList<OptionData | SearchQueryItem>
+            showLoadingPlaceholder={!areOptionsInitialized}
+            fixedNumItemsForLoader={4}
+            loaderSpeed={3}
```

Comment:
I almost wonder if we should make the `3` here some kind of var or default since that's what we want to consistently use across the app now?

Reply:
@daledah we can move the loader speed to a Const file under `TIMING` object.